src/providers/openstack/mock_tests.rs

```diff
@@ -51,3 +51,58 @@ fn test_ssh_keys_404_ok() {
     server.reset();
     provider.ssh_keys().unwrap_err();
 }
+
+#[test]
+fn test_instance_uuid() {
+    let mut server = mockito::Server::new();
+    let mut provider = OpenstackProviderNetwork::try_new().unwrap();
+    provider.client = provider.client.max_retries(0).mock_base_url(server.url());
+
+    server
+        .mock("GET", "/openstack/2012-08-10/meta_data.json")
+        .with_status(200)
+        .with_header("content-type", "application/json")
+        .with_body_from_file("tests/fixtures/openstack-metadata/openstack/2012-08-10/meta_data.json")
```

Comment:
This is failing the `cargo fmt` CI check.

Reply:
woops.